api/Pipfile

```
@@ -8,6 +8,8 @@ verify_ssl = true
 [packages]
 flask = "*"
 python-dotenv = "*"
+flask-migrate = "*"
+flask-sqlalchemy = "*"
 
```

`postgres` and `psycopg2` are the database adapters for SQLalchemy, and you need those added as well to the Pipfile. Here you go:
```suggestion
postgres = "*"
psycopg2 = "*"
```